### DISCUSSION TITLE
License page select/cancel button color poll

### DISCUSSION THREAD

**Discussion by bentshrimp** [AgainIoT/Open-Set-Go#58] [Polls]
The most selected one would be the one we use.

# 0
![Screenshot from 2023-10-23 20-30-21](https://github.com/AgainIoT/Open-Set-Go/assets/39232867/0be0d0ea-bebe-485b-968a-63067e0d28f2)

# 1
![Screenshot from 2023-10-23 20-30-42](https://github.com/AgainIoT/Open-Set-Go/assets/39232867/b681247c-34b8-4957-830e-6e49c3c715ea)

# 2 
![Screenshot from 2023-10-23 20-31-10](https://github.com/AgainIoT/Open-Set-Go/assets/39232867/160ffe1e-adb0-43b2-b8de-99911bf790dc)

# 3 
![Screenshot from 2023-10-23 20-32-17](https://github.com/AgainIoT/Open-Set-Go/assets/39232867/9ff534e1-699f-4ec1-ba71-aa0dac7396bd)

If you have other color combination recommendation, please leave comment.

**Comment by ymw0407** [comment]
When I look at the UX aspect, I think I should put focusing on what's chosen now. I don't think you have to insist on selecting/cancelling, but I think you should put focusing on selected side even if you go selected/selected. It's not about cancellation, it's about what you chose.
UX적인 측면을 보았을때 저는 focusing을 지금 선택된것에 놓아야한다고 생각합니다. 굳이 select/cancel을 고집하지 않고, select/selected를 가더라도 focusing을 selected쪽에 놓아야된다고 생각합니다. 취소가 중요한게 아니라, 결론적으로 무엇을 선택했는지가 중요하기 때문입니다.

**Reply by ymw0407** [reply]
In conclusion, you can cancel it!I think it's not important, it's more important to articulate that you chose this. In a typical case, there is no reason not to apply the license.
결론적으로 취소할 수 있다!가 중요한게 아니라, 너가 이걸 선택했다라는 것을 더 명확히 표현하는게 중요할 것 같습니다. 일반적인 경우에는, License 적용을 하지 않을 이유가 없기 때문입니다.

**Comment by Kim-Jiyun** [comment]
I agree with ymw0407.
So let's make the design number 1 and implement it by applying the color of the canceled state and the selected state in reverse!
저도 ymw0407님의 의견에 동의합니다.
그래서 디자인은 1번으로 하도록 하고 취소된 상태와 선택된 상태의 색깔을 반대로 적용하여 구현하겠습니다!

**Comment by ymw0407** [comment]
This discussion resolved at https://github.com/AgainIoT/Open-Set-Go_client/issues/159 !!!
